test/unit-test/FreeRTOS_IP_DiffConfig/FreeRTOS_IP_DiffConfig_utest.c

```diff
@@ -115,38 +115,38 @@ static size_t StubuxStreamBufferGetPtr_ReturnCorrectVals( StreamBuffer_t * pxBuf
     return ReleaseTCPPayloadBufferxByteCount;
 }
 
-static void vSetIPTaskHandle( TaskHandle_t xTaskHandleToSet )
-{
-    const uint8_t ucIPAddress[ ipIP_ADDRESS_LENGTH_BYTES ];
-    const uint8_t ucNetMask[ ipIP_ADDRESS_LENGTH_BYTES ];
-    const uint8_t ucGatewayAddress[ ipIP_ADDRESS_LENGTH_BYTES ];
-    const uint8_t ucDNSServerAddress[ ipIP_ADDRESS_LENGTH_BYTES ];
-    const uint8_t ucMACAddress[ ipMAC_ADDRESS_LENGTH_BYTES ];
+// static void vSetIPTaskHandle( TaskHandle_t xTaskHandleToSet )
```

Comment:
Can we remove if this test is not needed?

Reply:
sure